src/routes/(blog-article)/how-to-setup-the-development-environment/+page.md

```diff
@@ -86,28 +83,39 @@ sudo apt-get install libsqlite3-dev
 
 </CodeBlock>
 
+  <!-- Colocarle la razon de su uso migracion de datos -->
+
 ## Set Up the Torrust Tracker
 
 At the time of writing, the Torrust Tracker requires:
 
-- rustc 1.72.0-nightly (839e9a6e1 2023-07-02)
+- rustc 1.77.1-nightly (839e9a6e1 2024-03-26)
 
-You can run the Tracker with the following commands:
+Now , we will create the tracker service and inside it we will create the folders where the databases will be located:
 
 <CodeBlock lang="terminal">
 
-```s
-cd torrust-tracker/
-./bin/install.sh
-cargo run
+```bash
+  git clone https://github.com/torrust/torrust-tracker.git \
+  && cd torrust-tracker \
+  && cargo build --release \
+  && mkdir -p ./storage/tracker/lib/database \
+  && mkdir -p ./storage/tracker/lib/tls
 ```
 
 </CodeBlock>
```

Comment:
Hi @ivan9989-pkj I would use this:

```
<CodeBlock lang="terminal">

`` `terminal
git clone https://github.com/torrust/torrust-tracker.git \\
  && cd torrust-tracker \\
  && cargo build --release \\
  && mkdir -p ./storage/tracker/lib/database \\
  && mkdir -p ./storage/tracker/lib/tls
`` `

</CodeBlock>
```

Otherwise the `\` at the end of the line is not shown.